<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pip Matrix Resolver</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QTableView" name="requirementsView"/>
      <widget class="QTableView" name="matrixView"/>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="bottomSplitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QPlainTextEdit" name="logView">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <widget class="QMenu" name="menuFile">
    <property name="title"><string>&amp;File</string></property>
    <addaction name="actionOpenRequirements"/>
    <addaction name="actionFetchRequirements"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title"><string>&amp;Tools</string></property>
    <addaction name="actionCreateVenv"/>
    <addaction name="actionResolveMatrix"/>
    <addaction name="actionPause"/>
    <addaction name="actionResume"/>
    <addaction name="actionStop"/>
   </widget>
   <widget class="QMenu" name="menuBatch">
    <property name="title"><string>&amp;Batch</string></property>
    <addaction name="actionRunBatch"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title"><string>&amp;Help</string></property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuBatch"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle"><string>Main Toolbar</string></property>
   <attribute name="toolBarArea"><enum>TopToolBarArea</enum></attribute>
   <addaction name="actionOpenRequirements"/>
   <addaction name="actionFetchRequirements"/>
   <addaction name="actionCreateVenv"/>
   <addaction name="actionResolveMatrix"/>
   <addaction name="actionPause"/>
   <addaction name="actionResume"/>
   <addaction name="actionStop"/>
   <addaction name="actionRunBatch"/>
   <addaction name="actionAbout"/>
  </widget>

  <!-- Actions with cleaned iconsets -->
  <action name="actionOpenRequirements">
   <property name="icon">
    <iconset>
     <normaloff>:/resources/icons/open.svg</normaloff>
    </iconset>
   </property>
   <property name="text"><string>Open requirements file...</string></property>
  </action>
  <action name="actionFetchRequirements">
   <property name="icon">
    <iconset>
     <normaloff>:/resources/icons/url.svg</normaloff>
    </iconset>
   </property>
   <property name="text"><string>Fetch requirements from URL...</string></property>
  </action>
  <action name="actionExit">
   <property name="text"><string>Exit</string></property>
  </action>
  <action name="actionCreateVenv">
   <property name="icon">
    <iconset>
     <normaloff>:/resources/icons/venv.svg</normaloff>
    </iconset>
   </property>
   <property name="text"><string>Create/Update venv</string></property>
  </action>
  <action name="actionResolveMatrix">
   <property name="icon">
    <iconset>
     <normaloff>:/resources/icons/resolve.svg</normaloff>
    </iconset>
   </property>
   <property name="text"><string>Resolve matrix</string></property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset>
     <normaloff>:/resources/icons/pause.svg</normaloff>
    </iconset>
   </property>
   <property name="text"><string>Pause</string></property>
  </action>
  <action name="actionResume">
   <property name="icon">
    <iconset>
     <normaloff>:/resources/icons/resume.svg</normaloff>
    </iconset>
   </property>
   <property name="text"><string>Resume</string></property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset>
     <normaloff>:/resources/icons/stop.svg</normaloff>
    </iconset>
   </property>
   <property name="text"><string>Stop</string></property>
  </action>
  <action name="actionRunBatch">
   <property name="icon">
    <iconset>
     <normaloff>:/resources/icons/batch.svg</normaloff>
    </iconset>
   </property>
   <property name="text"><string>Run batch conversion to mp4</string></property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>:/resources/icons/info.svg</normaloff>
    </iconset>
   </property>
   <property name="text"><string>About</string></property>
  </action>
 </widget>
 <connections/>
</ui>
